hermes/validator/scorer_manager.py

```
@@ -57,22 +57,33 @@ async def compute_challenge_score(
             valid_scores = await asyncio.gather(
-                *(self.cal_ground_truth_score(ground_truth, r, cid_hash, token_usage_metrics, round_id=round_id) for r, _ in valid_miners)
+                *(score_with_semaphore(ground_truth, r, cid_hash, token_usage_metrics, round_id) for r, _ in valid_miners)
             )
         else:
             valid_scores = []
         
         # Reconstruct ground_truth_scores_raw in original order
         ground_truth_scores_raw = ["0.0"] * len(miner_synapses)
-        for (_, i), score in zip(valid_miners, valid_scores):
+        ground_truth_scores_error = [""] * len(miner_synapses)
+        for (_, i), (score, error) in zip(valid_miners, valid_scores):
```

Ruff (0.15.4)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)